assets/prototype/application/utilities/converters/number/toInteger.js

```diff
@@ -0,0 +1,13 @@
+export default function toInteger(dirtyNumber) {
```

Comment:
This is the same function as in `date-fns`: https://github.com/date-fns/date-fns/blob/master/src/_lib/toInteger/index.js

Reply:
Updated here: https://github.com/eventespresso/event-espresso-core/pull/2048/commits/f241f5249131d0697de8705454e6e4b2b20571c1

Reply:
And made a sanity check for these `if (dirtyNumber === null || dirtyNumber === true || dirtyNumber === false)`:
![image](https://user-images.githubusercontent.com/1477453/71107618-1654fb80-21ca-11ea-821b-8fcb573ae67b.png)
*** 
all good

Reply:
back to your question: I didn't find an exact or similar equivalent of `toInteger` in Ramda.